### PR TITLE
#112: Use `project-keeper`

### LINKED ISSUE #112: Use `project-keeper`
## Situation

Like in all our Java projects, we should use `project-keeper` to make the project uniform.

## Acceptance Criteria

1. Project keeper used.

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #112.

### CODE REVIEW

File: doc/changes/changes_3.4.0.md [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+## Bugfixes
+
+* #105: Fixed code smells.
 * #109: Adding "JDBC" as service in `withRequiredServices()` does not throw exception anymore.
 
+## Refactoring
+
+* #106: Introduced CI matrix test with Exasol 6.1.12 and 7.0.4
+* #112: Using `project-keeper` to manage common parts.
+
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_3.4.0.md:22]: the new dependency is missing

* Added `com.exaso.l:project-keeper-maven-plugin:0.4.1`

File: .travis.yml [hunk 2]
```diff
@@ -8,9 +8,9 @@ services:
   - docker
 
 env:
-  - DOCKER_DB_VERSION=7.0.2
```
Comment (chiaradiamarcelo) [inline .travis.yml:11]: why `7.0.2` is removed? what's the plan, having only one per Exasol major release?

File: doc/changes/changes_3.4.0.md (hunk 1 above)
Comment (chiaradiamarcelo) [reply doc/changes/changes_3.4.0.md:22]: Also, the new dependency is missing in the README file.

`| [Project Keeper Maven Plugin](https://github.com/exasol/project-keeper-maven-plugin)| Checking project structure                             | MIT License                   |`

Comment (redcatbear) [reply doc/changes/changes_3.4.0.md:22]: You are absolutely right.

File: .travis.yml (hunk 2 above)
Comment (redcatbear) [reply .travis.yml:11]: Yes. Latest per major. It is a trade-off between resource usage and coverage.

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**